### PR TITLE
add hikebike hillshading

### PR DESCRIPTION
**Pull request by ximex**

### CODE REVIEW

File: leaflet-providers.js [hunk 1]
```diff
@@ -495,9 +495,18 @@
...
+			url: 'http://{s}.tiles.wmflabs.org/{variant}/{z}/{x}/{y}.png',
 			options: {
-				attribution: '{attribution.OpenStreetMap}'
+				maxZoom: 19,
+				attribution: '{attribution.OpenStreetMap}',
+				variant: 'hikebike'
+			},
+			variants: {
+				HikeBike: {},
+				HillShadeing: {
```
Comment (jieter) [inline leaflet-providers.js]: typo here.

**Comment by jieter** [comment]
Thanks. Would be handy if you can put a link to the map in the PR description to new layers, saves me some time.

**Comment by jieter** [comment]
@ximex this cannot be merged, can you [rebase](http://git-scm.com/book/ch3-6.html)?

**Comment by ximex** [comment]
hmmm the rebasing didn't work as expected...
Link: http://www.hikebikemap.org/

**Comment by ximex** [comment]
should i make a new pull request?

**Comment by jieter** [comment]
Yes, or try tot rebase again...

**Pull request closed without merging**